runtime/sema/import.go

```diff
@@ -21,14 +21,15 @@ package sema
 import (
 	"github.com/onflow/cadence/runtime/ast"
 	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/common/orderedmap"
 )
 
 // Import
 
 type Import interface {
-	AllValueElements() *StringImportElementOrderedMap
+	AllValueElements() *orderedmap.OrderedMap[string, ImportElement]
```

Comment:
maybe define a type/alias for `*orderedmap.OrderedMap[string, ImportElement]`

Reply:
Good point! Added in [9045c19](https://github.com/onflow/cadence/pull/1804/commits/9045c1981258031fe0f9946dea9c4963b44d7e9d). That shaved off another ~100 LOC :-)